### PR TITLE
[CaaS | CIC | Kaby Lake | Pie] Adding permission for Health Monitorin…

### PR DESCRIPTION
**Pull request by taranpreetkaur0013**
…g Service: Storage Metrics

Porting Health Monitoring Service along with Storage HAL.

Tracked On: OAM-87516
Signed-off-by: tkaur <taranpreet.kaur@intel.com>

### CODE REVIEW

**Review by shyjumon-n — APPROVED** [review]
Required for health and storage HAL. 
Looks good to me, someone from NPI would also approve.

File: cic/cic.mk [hunk 1]
```diff
@@ -56,6 +56,10 @@ PRODUCT_PACKAGES += \
     Browser2 \
     audio.primary.$(TARGET_PRODUCT)
 
+PRODUCT_PACKAGES += health
+
```
Comment (hongcheng-xie) [inline cic/cic.mk:60]: Do we have health bin?  AFAIK, healthd bin was already included in product package by default

Comment (taranpreetkaur0013) [reply cic/cic.mk:60]: this is for addition of health HAL service, which was not enabled by default for CIC project. 
So, added the health service package android.hardware.health@2.0-service.intel here.

Comment (hongcheng-xie) [reply cic/cic.mk:60]: Good to know:)

**Review by hongcheng-xie — APPROVED** [review]

**Review by hongcheng-xie — APPROVED** [review]

**Pull request closed without merging**